packages/template-retail-react-app/package.json

```diff
@@ -104,7 +104,7 @@
     },
     {
       "path": "build/vendor.js",
-      "maxSize": "329 kB"
```

Comment:
We are increasing the bundle size +10kB because we are shipping OpenTelemetry libraries to the client, but since the trace is only measuring SSR I wonder if we can adjust the webpack config and use dynamic imports so these libs stay server-only. That way we keep tracing SSR but avoid the performance hit on the client